python/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -82,47 +82,65 @@ def test_is_correct_type(self):
     def test_matches_fixture(self):
         self.assertEqual(self._fixtures["info"], self._manager.info())
 
-class Test_setSettings(FixtureAugmentedTestCase):
+
+class Test_settings(FixtureAugmentedTestCase):
+    """
+    Check plugin's implementation of managerApi.ManagerInterface.settings.
+    """
+
+    def test_when_retrieved_settings_modified_then_newly_queried_settings_unmodified(self):
+        original = self._manager.settings()
+        expected = original.copy()
+        original["update"] = 123
+        self.assertEqual(self._manager.settings(), expected)
+
+
+class Test_initialize(FixtureAugmentedTestCase):
     """
     Check plugin's implementation of
-    managerApi.ManagerInterface.setSettings.
+    managerApi.ManagerInterface.initialize.
     """
     def setUp(self):
-        self.collectRequiredFixture('some_settings_with_valid_keys', skipTestIfMissing=True)
+        super(Test_initialize, self).setUp()
```

Comment:
Do we really need them? It says the default implementation is a no-op, and were not including them in other `setUp`/`tearDown` methods

Reply:
I added the `super` since it's not directly derived from `unittest.TestCase`, so the base class _could_ also override.  It doesn't though, so I can remove it.  I guess its a YAGNI situation at the moment.

Reply:
Also removed from `tearDown` in 1f16027e :roll_eyes:

Reply:
Ah yes, sorry I'd forgotten it wasn't just a direct derivation - good point. As long as you're happy with where were at now then 👍